EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: <to be assigned>
+title: Giving Ethereum fees to Future Salaries
+author: Victor Porton <porton@narod.ru>
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-02-13
+---
+
+## Simple Summary
+Transfer a part of Ethereum transfer/mining fees to Future Salaries contract
+
+## Abstract
+Part of mining/transfer fees to the contract `SalaryWithDAO` to incentivize traders to pay to common good producers. That directly benefits not only common good producers but miners/validators, too.
+
+## Motivation
+There are two motivations:
+
+1. It provides a big amount of "money" to common good producers. That obviously personally benefits common good producers, allowing them to live better human lives, it increases peoples' and organizations' both abilities and incentives to produce common goods. That benefits the humanity as a whole and the Ethereum ecosystem in particular. In my opinion (Victor Porton), this is crucial for survival of mankind because of the following reasons: a. Suppose an important scientific discovery was wrongly published and because of the wide-spread "no-republication of already published" scientific ethic it may become that there will be nothing in the mankind what would make this discovery to enter into scientific databases and/or raise reasonably high in search engines result pages. This would effectively mean that a part of science is missing in human knowledge (it could be rediscovered but not re-published due to the above described reasons). A missing part of the science would make the entire science development stuck. So, mankind would become unable to resist exestential problems (like climate change or asteroid impact) and likely die completely in near future. b. There is no other known (already discovered) economical incentive to finance climate actions than this proposal. So if this proposal won't be accepted, we are unable to resist climate change. You may assume that humanity has a way to overcome these problems and you are right: a way is this EIP.
+
+2. This would effectively decrease ETH supply. The necessity to decrease the ETH supply (by locking ETH in Future Salaries system for a long time) is a well-known important things to be done.
```

Comment:
Locking ETH doesn't decrease its supply.  At best it decreases *circulating supply*, which is a very different thing from total supply.

Reply:
Yes, I meant circulating supply, obviously. I will correct the wording.